src/main/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocument.java

```diff
@@ -923,6 +924,48 @@ protected boolean isCampusReviewRequired() {
 
         return false;
     }
+    
+    @Override
+    protected boolean isTaxReviewRequired() {
+        if (isPayeePurchaseOrderVendorHasWithholding()) {
+            return true;
+        }
+
+        String payeeTypeCode = this.getDvPayeeDetail().getDisbursementVoucherPayeeTypeCode();
+        if (payeeTypeCode.equals(KFSConstants.PaymentPayeeTypes.EMPLOYEE)) {
+            return false;
+        } else if (payeeTypeCode.equals(KFSConstants.PaymentPayeeTypes.VENDOR)) {
+            if(vendorService.isVendorInstitutionEmployee(this.getDvPayeeDetail().getDisbVchrVendorHeaderIdNumberAsInteger())){
```

Comment:
I know this was basically copied from the overridden method, but it makes me nervous that it's referencing the class variable rather than the getter. Since this service can't be injected, we're depending on the getter being called before this or we'll get an NPE. I would feel more comfortable with it calling the getter.